src/rust/bitbox02/src/backup.rs

```diff
@@ -51,3 +53,50 @@ pub fn check() -> Result<CheckData, Error> {
         err => Err(err),
     }
 }
+
+#[derive(Copy, Clone, Default)]
+struct Backup(bitbox02_sys::Backup);
+
+#[derive(Copy, Clone, Default)]
+struct BackupData(bitbox02_sys::BackupData);
+
+impl zeroize::DefaultIsZeroes for Backup {}
+impl zeroize::DefaultIsZeroes for BackupData {}
+
+pub struct RestoreData {
+    pub seed: zeroize::Zeroizing<Vec<u8>>,
+    // unix timestamp, UTC.
+    pub birthdate: u32,
+    // unix timestamp, UTC.
```

Comment:
as you know, unix timestamps do not depend on a timezone. a "unix timestamp, CET" would be very weird :)